nativelib/src/main/scala/java/lang/Class.scala

```diff
@@ -19,10 +19,15 @@ object rtti {
 }
 import rtti._
 
-/** @param rawty - Pointer with underlying Rt.Type info */
-final class _Class[A](val rawty: RawPtr) {
-  @alwaysinline private def ty: Ptr[Type] =
-    fromRawPtr[Type](rawty)
+final class _Class[A] {
+  var id: Int           = _
+  var traitId: Int      = _
+  var name: String      = _
+  var size: Int         = _
+  var idRangeUntil: Int = _
+
+  def isClass: Boolean =
```

Comment:
```suggestion
  private def isClass: Boolean =
```

Reply:
Even better, consider direction using `id < 0` in `isInterface()`, and use `!isInterface()` at the only other call site of `isClass`.